vpdq/cpp/pdq/README.md

```diff
@@ -0,0 +1,5 @@
+Vendorized PDQ CPP implementation
+
+PDQ is vendorized in order to ease the complexity of the VPDQ's build system and to avoid version conflicts.
+
+If VPDQ or PDQ are moved to their own repositories in the future, VPDQ's build system will still continue to function without modification.
```

Comment:
Ugh, hearing that submodules don't work if its to the same repo is the strongest argument I've heard yet to start breaking up the ThreatExchange repo.

Reply:
I think there is a way to do it, or with git subtrees, but that's beyond me. I would rather not ruin the repo doing some weird tricks.